src/cmd/library_add.js

```diff
@@ -88,7 +88,7 @@ export class LibraryAddCommand {
 
 	async addLibraryToProject(library) {
 		await this.site.addedLibrary(library.name, library.version);
-		await this.properties.addDependency(library.name, library.version);
+		return this.properties.addDependency(library.name, library.version);
```

Comment:
before the whenjs removal we used pipeline. According with the documentation this method always returns the result of the last task, in order to keep that behavior we need to return this.